pages/whats-new/2021-05-13-customerio-app-api-key.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: "Accessing Customer.io APP API"
+date: "2021-05-13"
+tags: ["improvement"]
+description: "Performing requests to the Customer.io App API"
+github: "https://github.com/grouparoo/grouparoo/pull/1770"
+---
+
+[Customer.io](https://customer.io/) separates their API into 2 the `TRACK API` and the `APP API` and you cannot access the Track API with the App API key nor access the App API with the track API key. In order to perform requests to the APP API required by the newest Sync Modes concept, Grouparoo now asks for the App API key.
```

Comment:
Some suggestions for grammatical clarity:

---
In order to <support all the available> Sync Modes[link to blog post], Grouparoo now asks for <both> the App API key <and Track API key in the Customer.io plugin>.  Customer.io separates their API into 2 <different parts,> the `TRACK API` and the `APP API`<.> <You> cannot access the Track API with the App API key<,> nor <can you> access the App API with the track API key.